bedrock/firefox/templates/firefox/features/private-browsing.html

```diff
@@ -42,6 +51,26 @@ <h2>{{ _('Browse without a trace') }}</h2>
       </div>
     </section>
 
+    {% if (variation == 'a') and (request.locale in ['en-US', 'en-GB', 'en-CA', 'en-AU']) %}
```

Comment:
Not a blocker, but you can specify locales in the `VariationTemplateView`, so you don't need to query locales directly in the template. I think for a short experimnet this is fine though.